mcweb/settings.py

```diff
@@ -76,6 +76,7 @@
     EMAIL_ORGANIZATION=(str, "Media Cloud Development"),
     GIT_REV=(str, ""),
     NEWS_SEARCH_API_URL=(str, "http://ramos.angwin:8000/v1/"),
+    PROVIDERS_TIMEOUT=(int, 60*10),
```

Comment:
I'd be surprised if this is sufficient (declares the typing and default, but doesn't fetch it into a variable)!